opslevel/resource_opslevel_service.go

```diff
@@ -295,8 +317,11 @@ func resourceServiceUpdate(d *schema.ResourceData, client *opslevel.Client) erro
 	if d.HasChange("tier_alias") {
 		input.Tier = d.Get("tier_alias").(string)
 	}
+	if d.HasChange("owner") {
```

Comment:
Should we do the same check as create here?  The way its coded now `owner_alias` will always override `owner` in an update but won't throw an error saying both fields shouldn't be used.  Maybe we extract whats in create to a method so it can be reused here?   While this works it might cause unexpected behaviour from the customer perspective.

Reply:
Yes! I will implement this.

Reply:
@rocktavious I added logic to ensure that on updates it will prevent the user from selecting both and having one override the other.

I don't think re-using the code for create and update by extracting it into its function was the way to go, since the function calls are slightly different.

---

Works on my machine ✅ 

Using both fields:

```
$ terraform apply
│ Error: can pass only one of: 'owner' or 'owner_alias'
│ 
│   with opslevel_service.new_service_test,
│   on service.tf line 5, in resource "opslevel_service" "new_service_test":
│    5: resource "opslevel_service" "new_service_test" {
```

Using one field:

```
$ terraform apply
# success....
```